dev/CommonStyles/InkToolbar_themeresources.xaml

```diff
@@ -272,23 +313,71 @@
+                                        <Setter Target="ExtensionGlyph.Opacity" Value="1" />
+                                    </VisualState.Setters>
+                                </VisualState>
+
+                                <VisualState x:Name="CheckedPressed">
+                                    <VisualState.Setters>
+                                        <Setter Target="RootElement.Background" Value="{ThemeResource InkToolbarButtonSelectedBackgroundPressed}" />
+                                        <Setter Target="Border.BorderBrush" Value="{ThemeResource InkToolbarButtonSelectedBorderBrushPressed}" />
+                                        <Setter Target="Content.Foreground" Value="{ThemeResource InkToolbarButtonSelectedForegroundPressed}" />
+                                        <Setter Target="CheckedContent.Foreground" Value="{ThemeResource InkToolbarButtonSelectedForegroundPressed}" />
+                                        <Setter Target="ExtensionGlyph.Foreground" Value="{ThemeResource InkToolbarAccentColorForegroundThemeBrush}" />
+                                        <Setter Target="Content.Opacity" Value="0" />
+                                        <Setter Target="CheckedContent.Opacity" Value="1" />
+                                        <Setter Target="SelectionAccent.Opacity" Value="0" />
+                                        <Setter Target="ExtensionGlyph.Opacity" Value="1" />
+                                    </VisualState.Setters>
+                                </VisualState>
+
+                                <VisualState x:Name="CheckedDisabled">
+                                    <VisualState.Setters>
+                                        <Setter Target="RootElement.Background" Value="{ThemeResource InkToolbarButtonSelectedBackgroundDisabled}" />
+                                        <Setter Target="Border.BorderBrush" Value="{ThemeResource InkToolbarButtonSelectedBorderBrushDisabled}" />
+                                        <Setter Target="Content.Foreground" Value="{ThemeResource InkToolbarButtonSelectedForegroundDisabled}" />
+                                        <Setter Target="CheckedContent.Foreground" Value="{ThemeResource InkToolbarButtonSelectedForegroundDisabled}" />
+                                        <Setter Target="ExtensionGlyph.Foreground" Value="{ThemeResource InkToolbarAccentColorForegroundThemeBrush}" />
+                                        <Setter Target="Content.Opacity" Value="0" />
+                                        <Setter Target="CheckedContent.Opacity" Value="1" />
+                                        <Setter Target="SelectionAccent.Opacity" Value="0" />
+                                        <Setter Target="ExtensionGlyph.Opacity" Value="1" />
                                     </VisualState.Setters>
                                 </VisualState>
 
+                            </VisualStateGroup>
```

Comment:
You can probably remove all this (and in the other places where they appear as well).

Reply:
I can not see here, are they related to selectionAccent?

Reply:
I mean the empty VisualStateGroups.